Tests/scripts/collect_tests/id_set.py

```diff
@@ -199,6 +200,8 @@ def __init__(self, marketplace: MarketplaceVersions) -> None:
 
             self.test_playbooks = self.id_to_test_playbook.values()
             self.modeling_rules = self.path_to_modeling_rule.values()
+            self.api_modules_to_integrations = {
+                script.object_id: script.imported_by for script in scripts if script.object_id.endswith("ApiModule")}
```

Comment:
I don't think you need the endswith condition. 
If you do it sounds like a bug in the graph